pytorch_blade/torch_blade/mlir/disc_engine_conversion.py

```diff
@@ -79,7 +79,7 @@ def _compile_torchscript(graph):
             # RUN: disc_compiler_main input_mlir_file.mlir output_file.so
             # redirect stdout to devnull
             subprocess.check_call(
-                [mhlo_compile_cmd, inp_mlir_file.name, out_file_name, "--multi-cc-support"],
+                [mhlo_compile_cmd, inp_mlir_file.name, out_file_name, "--multi-cc-support", "--mlir-elide-elementsattrs-if-larger=4"],
```

Comment:
nit: using some value larger than `8` is better to avoid eliding  some shape tensor value .

Reply:
done